api/system/v1alpha1/zone_insight.proto

```diff
@@ -23,6 +23,8 @@ message ZoneInsight {
 
   // Statistics about Envoy Admin Streams
   EnvoyAdminStreams envoy_admin_streams = 2;
+
+  bool authTokenProvided = 3;
```

Comment:
The name of variable doesn't seems to be constant with other fields: `auth_token_provided`.